pkgs/development/python-modules/ultralytics_thop/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, torch
+, pythonOlder
+, setuptools
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "ultralytics_thop";
+  version = "2.0.8";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
```

Comment:
You'd better fetch from their GitHub repo, to avoid future risk of problems similar to https://github.com/ultralytics/ultralytics/issues/18027 / https://github.com/advisories/GHSA-7x29-qqmq-v6qc